packages/block-editor/src/components/media-flow/index.js

```diff
@@ -0,0 +1,186 @@
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import {
+	FormFileUpload,
+	MenuItem,
+	Toolbar,
+	withNotices,
+} from '@wordpress/components';
+import { withSelect } from '@wordpress/data';
+import { compose } from '@wordpress/compose';
+
+/**
+ * Internal dependencies
+ */
+import BlockControls from '../block-controls';
+import MediaUpload from '../media-upload';
+import MediaUploadCheck from '../media-upload/check';
+import LinkEditor from '../url-popover/link-editor';
+import LinkViewer from '../url-popover/link-viewer';
+
+const MediaFlow = (
```

Comment:
I think there was a proposal to change the name of this component so that it's not quite as encompassing, is that still something that's needed?

Reply:
I don't remember such a proposal but I can change to any other name as needed :P)

Reply:
renamed to mediaReplaceFlow which is less encompassing and more accurate.